src/checkPassword.test.js

```diff
@@ -12,8 +12,52 @@ describe(`Function 'checkPassword':`, () => {
   });
 
   it(`should return 'true' for the valid password with 8 characters`, () => {
+    const result = checkPassword('Pass123!@');
 
+    expect(result).to.be.true;
+  });
+
+  
+  it('should return false for a password contains at least 1 digit, 1 special character, 1 uppercase letter.', () => {
+    const result = checkPassword('qwerty');
```

Comment:
not relevant test, your negative inputs should have only one deviation from the requirements